portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+
+async function getQuotes() {
+    const responseFromServer = await fetch('/welcome');
+    const quote = await responseFromServer.json();
+
+    
+
+    const word = quote[Math.floor(Math.random() * quote.length)];
+
+    const welcomeItems = document.getElementById('welcome-container');
```

Comment:
I think that if you're not going to stick with the use of list items, that it might be worth changing welcome-container from something other than a `<ul>`, maybe to a `<p>`